perun-core/src/test/java/cz/metacentrum/perun/core/api/AuthzResolverIntegrationTest.java

```diff
@@ -64,6 +64,378 @@ public void authorizedPerunAdmin() throws Exception {
 		assertTrue(AuthzResolver.authorized(session, "default_policy", Collections.emptyList()));
 	}
 
+	@Test
+	public void authorizedVoAdmin() throws Exception {
```

Comment:
Please add the System.out.println(CLASS_NAME + "The_actual_test_name"); to each test you created. I know there are many other tests which do not have that, we will fix that later.